### ISSUE TITLE
Focused icons in Library/Store black after hibernation

### ISSUE THREAD

**Issue by summershere84** [ValveSoftware/steam-for-linux#9042]
#### Your system information

* Steam client version (build number or date): Dec 15 2022 21:27:25
* Distribution (e.g. Ubuntu): Debian
* Opted into Steam client beta?: [Yes/No] Yes
* Have you checked for system updates?: [Yes/No] Yes

#### Please describe your issue in as much detail as possible:
Describe what you _expected_ should happen and what _did_ happen. Please link any large code pastes as a [Github Gist](https://gist.github.com/)

It's a minor issue, but mousing over icons in the Library or Store go black (i.e. doesn't load the icon) when they are supposed to get slightly bigger and have the pop up with more details about the game to the side.  This only happens after system hibernate, and restarting the steam client fixes it.  There's no functionality lost, it's just kindof annoying.

There are some issues with videos too, if there's a video going on (i.e. Live or looking at a store page and videos autoplay), a good portion of the client goes black below the video and text is not rendered. Scrolling makes it even weirder.

Basically, rendering within the client is mildly broken after the OS has hibernated.  Also this has been true for a while (perhaps a year? or more?) I just haven't filed a ticket on it previously.  It's quite minor and easy to live with, but it would be nice if it got fixed.

#### Steps for reproducing this issue:

1.  Start Steam Client.
2.  Hibernate the OS.
3.  Restart

**Comment by kisak-valve** [comment]
Related: #8584
